theme/mathform.css

```diff
@@ -37,6 +37,8 @@
 
 		& .ck-input {
 			width: 100%;
+			/* without min-with, balloon-panel doesn't respect width of textarea */
+			min-width: 0;
```

Comment:
I don't get it, what exactly is the issue we're solving here? The only difference I'm seeing is that `min-width: 0` allows me to resize textarea to 0 which looks bad.

Reply:
Without `min-width`, it looks like this for me:
<img width="674" alt="Zrzut ekranu 2023-12-13 o 07 26 11" src="https://github.com/bethinkpl/ckeditor5-math/assets/2725894/6aa849b0-7db3-4b83-b3be-807fcd72120f">

Reply:
Interesting, it works correctly for me without `min-width`:
![Screenshot 2023-12-13 at 08 37 08](https://github.com/bethinkpl/ckeditor5-math/assets/7030884/1652e365-d781-41b7-be07-d5d0bc4fc24c)

Can we at least change it to something bigger than 0 to avoid the following look?
![Screenshot 2023-12-13 at 08 35 53](https://github.com/bethinkpl/ckeditor5-math/assets/7030884/bf0be87c-a67d-4bc8-80b3-1205884efaba)

Reply:
> it works correctly for me without min-width

Did you disable this style in dev console or removed from the code? Balloon width is calculated based on width of the content on show, so disabling styles in console doesn't give reliable effect...

Reply:
Installed a version without `min-width` in ckeditor5-bethink and it works.